src/components/ProfileAvatar.tsx

```diff
@@ -0,0 +1,229 @@
+import Skeleton from '@mui/material/Skeleton'
+import Slider from '@mui/material/Slider'
+import VerifiedBorder from 'assets/VerifiedProfilePic.svg'
+import React, { useEffect, useState } from 'react'
+import { Button } from '@mui/material'
+import { Modal } from 'react-bootstrap'
```

Comment:
Copied most of this code from the previous Profile page (where this functionality was implemented originally).  Changing this Modal to a MUI Dialog is tricky because of all of the custom css being applied for layout (the Cropper has absolute positioning, and is a bit difficult to integrate).